src/main/groovy/jenkins/automation/builders/JsJobBuilder.groovy

```diff
@@ -4,8 +4,31 @@ import javaposse.jobdsl.dsl.DslFactory
 import javaposse.jobdsl.dsl.Job
 import jenkins.automation.utils.ScmUtils
 
-class JsJobBuilder {
+/**
+ * @param name          Job name
+ * @param description   Job description
+ * @param pollScmSchedule  Optional Configure Jenkins to poll changes in SCM. Cron style schedule string.
+ * @param artifacts     Optional Closure defaulted to closure below :
+                                  {
+                                     pattern("dist/")
+                                     fingerprint()
+                                     defaultExcludes()
+
+                                     }
+
+ * @param emails  List or String of notification email addresses
+ * @param repos List of repos to watch
+ * @param use_versions flag to check out the repo at a specific tag. The tag is parsed out from url property appended after '@' sign.
```

Comment:
This one should probably make note of the fact that it only applies in multi-SCM situations.